src/Translations/MultiFileLoader.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the hyn/multi-tenant package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://laravel-tenancy.com
+ * @see https://github.com/hyn/multi-tenant
+ */
+
+namespace Hyn\Tenancy\Translations;
+
+use Illuminate\Translation\FileLoader;
+use Illuminate\Contracts\Translation\Loader;
+use Illuminate\Filesystem\Filesystem;
+
+class MultiFileLoader extends FileLoader
+{
+    /**
+     * @var \Illuminate\Translation\FileLoader
+     */
+    protected $fileLoaders = [];
+
+    /**
+     * Create a new file loader instance.
+     *
+     * @param  \Illuminate\Filesystem\Filesystem  $files
+     * @param  string  $path
+     * @return void
+     */
+    public function __construct(Filesystem $files, $path)
+    {
+        parent::__construct($files, $path);
```

Comment:
There's no need to define the construct method if all it does is proxy to the parent.